Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:25 as builder
+
+RUN dnf install -y \
+    git-core \
+    gcc \
+    gcc-c++ \
+    boost-devel \
+    cmake \
+    autoconf \
+    automake \
+    libtool \
+    mariadb-devel \
+    file \
+    hostname \
+    mariadb-server
+
+RUN git clone --branch master --single-branch --depth 1 https://github.com/anton-povarov/meow /_src/meow
```

Comment:
@anton-povarov It would be nice to create initial release so we could lock on it and use `ADD ...tar.gz` as well.